ruleset.xml

```diff
@@ -1,23 +1,5 @@
 <?xml version="1.0"?>
 <ruleset name="Zend Framework Coding Standard">
     <description>Zend Framework Coding Standard</description>
-
-    <!-- display progress -->
-    <arg value="p"/>
-    <arg name="colors"/>
-
-    <!-- inherit rules from: -->
-    <rule ref="PSR2"/>
-    <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
-    <rule ref="Generic.Formatting.SpaceAfterNot"/>
-    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-        <properties>
-            <property name="ignoreNewlines" value="true"/>
-        </properties>
-    </rule>
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
-        <properties>
-            <property name="ignoreBlankLines" value="false"/>
-        </properties>
-    </rule>
+    <rule ref="vendor/zendframework/zend-coding-standard/ZendCodingStandard/ruleset.xml"/>
```

Comment:
> In components that already use this package, this particular file is the one referenced. As an example, from zendframework/zend-expressive... Shouldn't this then be the full ruleset, and not reference another ruleset?

As you say, in expressive, this file is imported: `vendor/zendframework/zend-coding-standard/ruleset.xml`.
However for PHP_CodeSniffer to work, a ruleset.xml file must be `some/path/<ruleset_name>/ruleset.xml`. I've tried several options and even including this in `src` directly. This is the only way PHP_CodeSniffer detects the ruleset as a valid ruleset. I also tried this with PHP_CodeSniffer 3.0RC with the same result.

I guess using twice the name ruleset.xml is confusing. The first (this) one includes the real standard ruleset. I guess including './vendor/zendframework/zend-coding-standard/src/ZendCodingStandard/ruleset.xml' might also work, but than all components need another PR.

> The referenced path is invalid for this repository (should be src/ZendCodingStandard/ruleset.xml), though I'm not sure if that's a problem or not.

Since this ruleset.xml file is only meant for inclusion in other projects, it doesn't matter. Running phpcs inside this project will use phpcs.xml.

I have tested this against another [repo](https://github.com/xtreamwayz/zend-coding-test) and it worked as it is. I'll test zf components later on.